doc/nrf/known_issues.rst

```diff
@@ -95,7 +103,7 @@ The issues in this section are related to :ref:`ug_ble_controller`.
 
 .. _ncsdk_19865:
 
-.. rst-class:: v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0
```

Comment:
Hi @jori-nordic
I think this one is already fixed and should not be tagged for NCS 2.4. Am I right?

Reply:
I think it's fixed yes